### PR TITLE
code review

### PR DESCRIPTION
**Pull request by stanohlivjak**
pull requst

### CODE REVIEW

**Comment by lukasorgovan** [comment]
You have a conflict here :) please resolve conflict.

File: Solution.class [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Solution {
+    public boolean isPalindrome(int x) {
+        boolean result = false;
+        String temp = String.valueOf(x);
```
Comment (lukasorgovan) [inline Solution.class:4]: I am no Java expert, but what about using String builder class to create something 
https://www.javatpoint.com/StringBuilder-class

stringHalf = 1/2 string
secondHalf = 2/2 string - reversed
stringHalf === secondHalf  

of course you need to check odd numbered length of string :) just an idea.

**Comment by stanohlivjak** [comment]
The conflict should be resolved.

**Review by lukasorgovan — CHANGES_REQUESTED** [review]
ok you are trying to remove README.md file, not sure why, but you should not be removing this file :) 
It is there to help you guys remember basic stuff :)

**Comment by stanohlivjak** [comment]
I hope that everything is fixed now. :)

**Pull request merged**